src/main/java/com/noticemedan/map/model/KDTree/ForestCreator.java

```diff
@@ -0,0 +1,29 @@
+package com.noticemedan.map.model.KDTree;
+
+import com.noticemedan.map.model.MapObject;
+import com.noticemedan.map.model.MapObjectCreater;
+import lombok.Getter;
+
+import java.awt.*;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
```

Comment:
Brug så vidt muligt Vavr's collections (List, Map, Set osv), for konsistensens skyld

Reply:
Hahahahahaha konsistensen?? xD

Reply:
@emilbartholdy  det er ikke det store at lave det om da det kun er typer. Det er bare at tage dem en af gangen og se om compileren brokker sig